test/tests/functional/pbs_dup_acc_log_for_resv.py

```diff
@@ -142,46 +193,82 @@ def test_standing_reservation(self):
         else:
             self.logger.info('Missing timezone, using America/Los_Angeles')
             tzone = 'America/Los_Angeles'
+        if _m == PTL_API:
+            self.server.set_op_mode(PTL_API)
+        instance1_start = int(time.time() + 5)
+        instance1_end = int(time.time() + 35)
         a = {'Resource_List.select': '1:ncpus=1',
              ATTR_resv_rrule: 'FREQ=MINUTELY;COUNT=2',
              ATTR_resv_timezone: tzone,
-             ATTR_resv_standing: '1',
-             'reserve_start': time.time() + 5,
-             'reserve_end': time.time() + 60, }
+             'reserve_start': instance1_start,
+             'reserve_end': instance1_end}
         r = Reservation(TEST_USER, a)
         rid = self.server.submit(r)
-        a = {'reserve_state': (MATCH_RE, "RESV_RUNNING|5")}
-        self.server.expect(RESV, a, id=rid, max_attempts=30, offset=6)
-        if _m == PTL_API:
-            self.server.set_op_mode(PTL_API)
+        a = {'reserve_state': (MATCH_RE, "RESV_RUNNING|5"),
+             'reserve_index': (MATCH_RE, "1")}
```

Comment:
I don't think you need to match RE on this one.  You're just matching the string of "1"